internal/validator/main.go

```diff
@@ -0,0 +1,144 @@
+package main
```

Comment:
If I understand it correctly, this is supposed to be a ci tool. Shouldn't we have this at `cmd/validator/main.go` instead of `internal`?